example/ios/iOS UI Test/iOS UI Test/MCTMsgViewController.mm

```diff
@@ -257,19 +257,35 @@ - (NSData *) _convertToJPEGData:(NSData *)data {
     CGImageDestinationRef destination;
     NSMutableData * destData = [NSMutableData data];
 
-    destination = CGImageDestinationCreateWithData((__bridge CFMutableDataRef) destData,
-                                                   (CFStringRef) @"public.jpeg",
-                                                   1, NULL);
-    
-    CGImageDestinationAddImage(destination, thumbnail, NULL);
-    CGImageDestinationFinalize(destination);
-
-    CFRelease(destination);
+    if (destData != nil) {
+        destination = CGImageDestinationCreateWithData((__bridge CFMutableDataRef) destData,
+                                                       (CFStringRef) @"public.jpeg",
+                                                       1, NULL);
+        if ((destination != NULL) && (thumbnail != NULL)) {
```

Comment:
destination is likely to be always valid. I can't see a reason why it would be NULL.
(maybe memory exhaustion but terrible things would happen before then, we can just ignore that case)